app/views/foreman_tasks/layouts/react.html.erb

```diff
@@ -9,4 +12,4 @@
   <div id="foremanTasksReactRoot"></div>
 <% end %>
 <%= render file: "layouts/base" %>
-<%= mount_react_component('ForemanTasks', '#foremanTasksReactRoot',{ :flatten_data => true }) %>
+<%= mount_react_component('ForemanTasks', '#foremanTasksReactRoot') %>
```

Comment:
why is it still needed after adding react-router?

Reply:
To load the router

Reply:
Ok, pretty soon after https://github.com/theforeman/foreman/pull/6936 will get merge this won't be needed anymore.